release/ray_release/glue.py

```diff
@@ -210,6 +211,13 @@ def run_release_test(
         cluster_timeout = int(
             test["run"].get("session_timeout", DEFAULT_CLUSTER_TIMEOUT)
         )
+        prepare_cmd = test["run"].get("prepare", None)
+        if prepare_cmd:
+            prepare_timeout = test["run"].get("prepare_timeout", command_timeout)
+        else:
+            prepare_timeout = 0
+        command_and_prepare_timeout = command_timeout + prepare_timeout
+
```

Comment:
what is happening here? sorry but it is hard to understand context without a comment